project-base/config/paths.yaml

```diff
@@ -1,6 +1,5 @@
 parameters:
     shopsys.content_dir_name: 'content'
-    shopsys.css_version_filepath: '%shopsys.web_dir%/assets/cssVersion'
```

Comment:
this is not related to a scope of this PR and I suggest to remove the change to avoid confusion

Reply:
That's fair point